CHANGELOG.md

```diff
@@ -7,11 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+* **Dynamic Schema** Refactored the carousel to have a dynamic schema.
```

Comment:
You should decide whether the changelog will use the past tense or the infinitive.
https://github.com/vtex-apps/shelf/blob/master/CHANGELOG.md

Reply:
👍